docs/firmware/fw.md

```diff
@@ -58,3 +58,18 @@ following documents.
 
 [FW_JUMP]: fw_jump.md
 [FW_PAYLOAD]: fw_payload.md
+
+Providing different payloads to OpenSBI Firmware
+------------------------------------------------
+OpenSBI firmware can accept various payloads as an compile time options.
+Typically, these payload refer to the next stage boot loader (e.g. U-Boot)
+or supervisor operating system images (e.g. Linux). By default, OpenSBI
+firmware includes a test payload in absence of FW_PAYLOAD_PATH argument.
+
+To provide a payload during compile time
```

Comment:
To specify a payload at compile time, the make variable *FW_PAYLOAD_PATH* is used.